bears/vcs/git/GitCommitBear.py

```diff
@@ -27,7 +27,7 @@ def run(self,
             shortlog_trailing_period: bool=None,
             shortlog_imperative_check: bool=True):
         """
-        Checks the current git commit message at HEAD.
+        Check the current git commit message at HEAD.
```

Comment:
Are we doing this everywhere ? Can we use pydocstyle then ?
Loads of places where this is not imperative